### PR TITLE
Update JFrog CLI to 2.52.2 and update dependencies

### PR DESCRIPTION
**Pull request by sverdlov93**
- [ ] All [tests](https://github.com/jfrog/jfrog-azure-devops-extension#testing) passed. If this feature is not already covered by the tests, I added new tests.
- [ ] This pull request is on the dev branch.
- [ ] I used `npm run format` for formatting the code before submitting the pull request.
-----

### CODE REVIEW

**Comment by ashokm** [comment]
Looks like newer versions have been [released](https://github.com/jfrog/jfrog-cli/releases).
Please update to the latest released version (**2.49.1** at time of writing).

**Comment by sverdlov93** [comment]
> Looks like newer versions have been [released](https://github.com/jfrog/jfrog-cli/releases). Please update to the latest released version (**2.49.1** at time of writing).
@ashokm 
Yes, that's the reason we waited. We will release it soon.

**Comment by ashokm** [comment]
Are there any ETAs on a release?

File: buildScripts/publish-private.sh [hunk 1]
```diff
@@ -19,15 +20,22 @@ if [ -z "$ADO_ARTIFACTORY_API_KEY" ]; then
...
-npx tfx extension unpublish -t $ADO_ARTIFACTORY_API_KEY --extension-id jfrog-azure-devops-extension --publisher $PUBLISHER
-npx tfx extension create --manifest-globs vss-extension-private.json --publisher $PUBLISHER
-npx tfx extension publish -t $ADO_ARTIFACTORY_API_KEY --publisher $PUBLISHER --manifests vss-extension-private.json --override "{\"public\": false, \"version\": \"$RANDOM.$RANDOM.$RANDOM\", \"description\": \"Commit SHA: $GIT_HEAD\"}" --share-with $ADO_ARTIFACTORY_DEVELOPER
-npx tfx extension install --publisher $PUBLISHER --extension-id jfrog-azure-devops-extension --service-url https://$ADO_ARTIFACTORY_DEVELOPER.visualstudio.com -t $ADO_ARTIFACTORY_API_KEY
+npx tfx extension unshare -t "$ADO_ARTIFACTORY_API_KEY" --extension-id jfrog-azure-devops-extension --publisher "$PUBLISHER" --unshare-with "$ADO_ARTIFACTORY_DEVELOPER" 2>/dev/null
+npx tfx extension unpublish -t "$ADO_ARTIFACTORY_API_KEY" --extension-id jfrog-azure-devops-extension --publisher "$PUBLISHER"
+npx tfx extension create --manifest-globs vss-extension-private.json --publisher "$PUBLISHER"
+# Check that vsix size is less then 30MB
+vsixSize="$(du -m -- *.vsix | awk '{print $1}')"
```
Comment (yahavi) [inline buildScripts/publish-private.sh]: Ensure that only the first one tooked
```suggestion
vsixSize="$(du -m -- *.vsix | awk '{print $1}' | head -1)"
```

**Review by yahavi — CHANGES_REQUESTED** [review]

**Pull request merged**